samples/msal-node-samples/nextjs-rsc-ts/src/serverConfig.ts

```diff
@@ -0,0 +1,32 @@
+// https://nextjs.org/docs/getting-started/react-essentials#the-server-only-package
+// importing server-only as this module contains secrets that should not be exposed to the client
+import { Configuration } from "@azure/msal-node";
+import "server-only";
+
+export const graphConfig = {
+  meEndpoint: "https://graph.microsoft.com/v1.0/me",
+  eventEndpoint: "https://graph.microsoft.com/v1.0/me/calendar/events?$top=1",
+  profilePhotoEndpoint: "https://graph.microsoft.com/v1.0/me/photo/$value",
+};
+
+export const msalConfig: Configuration = {
+  auth: {
+    clientId: process.env.AZURE_AD_CLIENT_ID!,
+    clientSecret: process.env.AZURE_AD_CLIENT_SECRET!,
+    authority:
+      process.env.AZURE_AD_AUTHORITY ??
+      "https://login.microsoftonline.com/common",
+  },
```

Comment:
can we add logging as well? https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/logging.md